products/handlers/createProduct.js

```diff
@@ -0,0 +1,69 @@
+import AWS from "aws-sdk";
+import { v4 as uuid } from 'uuid';
+
+const corsHeaders = {
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Credentials': true,
+  'Content-Type': 'application/json'
+}
+const error = ({code, err, data}) => {
+  console.log('Something went wrong. Error: ' + err, data);
+
+  return {
+    statusCode: code || 500,
+    isBase64Encoded: false,
+    headers: corsHeaders,
+    body: 'Something went wrong. Error: ' + err,
+  }
+}
+
+const props = ['count', 'title', 'description', 'price'];
+
+const createProduct = async (event) => {
+  console.log(event);
+
+  const itemId = uuid();
+  const body = JSON.parse(event.body);
+  const dynamo = new AWS.DynamoDB.DocumentClient();
+
+  try {
+    const missedFromSchema = props.find(prop => !body.hasOwnProperty(prop));
```

Comment:
There are validation libs like Joi, zod etc, if you want to use something written specifically for validation.